editor/import/resource_importer_obj.cpp

```diff
@@ -378,12 +380,13 @@ static Error _parse_obj(const String &p_path, List<Ref<Mesh>> &r_meshes, bool p_
 					surf_tool->generate_normals();
 				}
 
+				// Force triangle pairs to share tangents
```

Comment:
```suggestion
				// Force triangle pairs to share tangents.
```
And same for other comments, just code style nitpick